ConcernsCaseWork/ConcernsCaseWork/Properties/launchSettings.json

```diff
@@ -23,7 +23,7 @@
         "ASPNETCORE_ENVIRONMENT": "Development",
         "ASPNETCORE_HOSTINGSTARTUPASSEMBLIES": "Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation"
       },
-      "applicationUrl": "https://localhost:5001;http://localhost:5000"
+      "applicationUrl": "https://localhost:5003;http://localhost:5002"
```

Comment:
IDK if we should maybe have this as default port as it's a UI?

Reply:
Yes this was unintentional, I have to change them on my local machine